web/src/constants/Analyzer.constants.ts

```diff
@@ -0,0 +1,8 @@
+export const ERROR_HEADER: Record<string, string> = {
+  attribute_naming_error: 'The following attributes do not adhere to the naming convention:',
```

Comment:
Should we generate this from the BE? this could be useful to have as part the CLI as well, so it is not duplicated